index.js

```diff
@@ -7,7 +7,7 @@ const port=process.env.PORT||3000;
 app.set('view engine', 'hbs');
 app.use('/bower_components',  express.static(__dirname + '/bower_components'));
 app.use(express.static(path.join(__dirname, 'public')));
-
+app.use(express.static(path.join(__dirname, 'media')));
 
 app.get('/',(req,res)=>{
```

Comment:
Can you please add a route to the resource.hbs in the index

Reply:
sure, done